core/arch/arm/plat-sam/matrix.h

```diff
@@ -31,6 +31,44 @@
 
 #include <tee_api_types.h>
```

Comment:
include also _stdint.h_ adn _types_ext.h_ for `uint32_t`, `vaddr_t` and `paddr_t` types.

Reply:
Updated to 
```
#include <stdint.h>
#include <tee_api_types.h>
#include <types_ext.h>
```